### DISCUSSION TITLE
Show best prices from other countries using eshop prices

### DISCUSSION THREAD

**Discussion by cohako** [stephannv/nindika#200] [Ideas]
Hello
What if we could see the best prices (limited to 3 or 5) for a game using all countries/regions. Like eshop-prices do.

This way the user could snatch good prices outside Brazil.

What do you think?

**Comment by stephannv** [comment]
Hi, @cohako . Thanks for the suggestion.

The nindika's main goal is help users to discover games. To price comparison between countries I think eshop prices already does a excellent job.
 There are more reasons to not add other countries to nindika:
- Brazil is already one of the most cheapest eshops, and sometimes isn't worthy change account country or pay the taxes (IOF, Spread).
- Nintendo, in some countries, is blocking purchases from credit cards issued outside the origin country, so it's frustating when you see a good price but cannot complete the checkout.
- Adding more countries will bring complexity to importation task. I already worked on a eshop prices-like website (SaveCoins) and it was hard to import and link data between countries, because there are three main API (america, europe, japan) with different ids (eg. mario odyssey id is one in america api, other in europe api).

So, the next features I'm planning to nindika is related to game discoverability and not about prices.
